retropie-shell-script-boilerplate.sh

```diff
@@ -1,33 +1,45 @@
 #!/usr/bin/env bash
 
-# [SCRIPT_TITLE] (e.g. Retropie Shell Script Boilerplate.)
+# [SCRIPT_NAME] (e.g.: script_template.sh)
 # [SCRIPT_DESCRIPTION] (e.g. A template for building shell scripts for RetroPie.)
 #
 # Author: [AUTHOR] (e.g. hiulit)
-# If you have the script in a GitHub Repository, use the comments below. If not, just remove them.
-# Repository: [REPOSITORY_URL] (e.g. https://github.com/hiulit/RetroPie-Shell-Script-Boilerplate)
-# Issues: [REPOSITORY_ISSUES_URL] (e.g. https://github.com/hiulit/RetroPie-Shell-Script-Boilerplate/issues)
+# Repository: [REPO_URL] (e.g. https://github.com/hiulit/RetroPie-Shell-Script-Boilerplate)
```

Comment:
I don't see the benefits of abbreviating REPOSITORY to REPO. But that's personal taste :P

Reply:
Sometimes I use the repo URL on the script (usually on a update procedure). And"repo" seems to be a very well accepted contraction for repository. That being said, I usually use `REPO_URL` as a variable name as well

(who wants to type `REPOSITORY_URL` here and there in the code?! :D)

Reply:
OK, you're right! It's a waste of time, typing REPOSITORY_URL instead of REPO_URL and arguing about it :D